### PR TITLE
fix(wallet/frontend): update account settings page

### LINKED ISSUE #1674: [BUG] Error on saving profile details
### Steps to reproduce

1. Login into your account 
2. Go to Settings
3. Click edit to edit your account
4. Change/update your name
5. Click on save profile

### Expected result

Userc. profile name update should save

### Actual result

User gets an `Requested path / was not found` error message and the details are not saved

### Screenshots or videos

Screenshot 2024-10-02 at 14 57 10

### Additional context

_No response_

### Operating system

macOS

### Operating system version

_No response_

### Browsers

Chrome

### Browser version

_No response_

### PR DESCRIPTION
**Pull request by adrianboros**
### Context

- fixes #1674

### CODE REVIEW

**Review by Tymmmy — CHANGES_REQUESTED** [review]
Check comment for changes

**Review by Tymmmy — APPROVED** [review]

**Review by Tymmmy — CHANGES_REQUESTED** [review]
Remove updateProfile as well

File: packages/wallet/frontend/src/lib/api/user.ts [hunk 1]
```diff
@@ -117,10 +117,6 @@ type VerifyEmailResponse = SuccessResponse | VerifyEmailError
 type MeResult = SuccessResponse<UserResponse>
 type MeResponse = MeResult | ErrorResponse
 
-type ProfileArgs = z.infer<typeof profileSchema>
```
Comment (Tymmmy) [inline packages/wallet/frontend/src/lib/api/user.ts:120]: is profileschema removed?

Comment (adrianboros) [reply packages/wallet/frontend/src/lib/api/user.ts:120]: now yes

**Review by Tymmmy — APPROVED** [review]

**Pull request merged**